.changelog/2684.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_organization: Avoids provider produced inconsistent result when `USER_NOT_FOUND`
```

Comment:
```suggestion
resource/mongodbatlas_organization: Avoids inconsistent result 
returned by provider when `USER_NOT_FOUND`
```

Reply:
Thank you updated in 811add510b9a24b40263de6bfcdd298593445bb4
(couldn't apply suggestion since it has a newline)